Plugin/Gateway.php

```diff
@@ -103,4 +103,24 @@ public function getIdealIssuers() {
 			return null;
 		}
 	}
+
+	/**
+	 * change api response error to userfriendly error message
+	 *
+	 * @param responseError from api
+	 *
+	 * @return String
+	 */
+	public function getUserFriendlyError($responseError = '') {
+		switch ($responseError) {
+			case 'Not subscribed to money transfer service':
+			case 'API key not valid':
+				$newMessage = _('Can\'t use this payment method, please try a different method.');
```

Comment:
I think we could change this to `"Can't use this payment method, please try a different method."`. Would make the translations more readable, also in the translation files.

Reply:
updated it :+1: